xml/deploy_cephadm.xml

```diff
@@ -518,16 +518,35 @@ o- ssh ................................................... [Key Pair set]
     <para>
      Select one of the &sminion;s to be a time server for the rest of the
      cluster, and configure it to synchronize its time with a reliable time
-     source(s). Verify that the time synchronization service is enabled on each
-     cluster node on system start-up.
+     source outside of the cluster.
     </para>
 <screen>
 &prompt.smaster;ceph-salt config /time_server/server_hostname set ses-master.example.com
-&prompt.smaster;ceph-salt config /time_server/external_servers add 0.pt.pool.ntp.org
+&prompt.smaster;ceph-salt config /time_server/external_servers add pool.ntp.org
+</screen>
+    <para>
+     Verify that the time synchronization service is enabled on each cluster
+     node on system start-up:
+    </para>
+<screen>
+&prompt.sminion;systemctl enable chronyd.service
```

Comment:
I don't see a reason to ask the user to manually enable the `chrony` service on all nodes.

Note that this will be done automatically by ceph-salt [1] when you latter apply the config by running (`ceph-salt apply`).

[1] https://github.com/ceph/ceph-salt/blob/master/ceph-salt-formula/salt/ceph-salt/time-prep.sls#L40